### DISCUSSION TITLE
How can I change date type to relative date in theme?

### DISCUSSION THREAD

**Discussion by alerezaaa** [transitive-bullshit/nextjs-notion-starter-kit#415] [Q&A]
Hi there
I use "relative date" instead on "month and time" in my Notion database but it is shown as full-date in this theme
How can I change the theme codes to show relative date for posts date?
I think its related to this line of code: [link](https://github.com/alerezaaa/nextjs-notion-starter-kit/blob/afbab1917fd793eb08729da56700e541ce3b4ee1/components/NotionPage.tsx#L108) But I dont know how to solve it
Thanks in advance

**Comment by alerezaaa** [comment]
nothing??

**Comment by oeyoews** [comment]
> Hi there
> I use "relative date" instead on "month and time" in my Notion database but it is shown as full-date in this theme
> How can I change the theme codes to show relative date for posts date?
> I think its related to this line of code: [link](https://github.com/alerezaaa/nextjs-notion-starter-kit/blob/afbab1917fd793eb08729da56700e541ce3b4ee1/components/NotionPage.tsx#L108) But I dont know how to solve it
> Thanks in advance

[format](https://github.com/NotionX/react-notion-x/blob/3aef81f18d79dfa5c86a27bf3934d13c77664323/packages/notion-utils/src/format-date.ts#L3)

**Reply by alerezaaa** [reply]
> > Hi there
> > I use "relative date" instead on "month and time" in my Notion database but it is shown as full-date in this theme
> > How can I change the theme codes to show relative date for posts date?
> > I think its related to this line of code: [link](https://github.com/alerezaaa/nextjs-notion-starter-kit/blob/afbab1917fd793eb08729da56700e541ce3b4ee1/components/NotionPage.tsx#L108) But I dont know how to solve it
> > Thanks in advance
> 
> [format](https://github.com/NotionX/react-notion-x/blob/3aef81f18d79dfa5c86a27bf3934d13c77664323/packages/notion-utils/src/format-date.ts#L3)

so it seems that I cant change this date style in my fork

thanks

**Reply by oeyoews** [reply]
https://github.com/oeyoews/nextjs-notion-starter-kit/commit/d2123fe5ae8ddc9cc7f066f110535b63075481fe
I tested it, it doesn't seem to work

**Reply by alerezaaa** [reply]
> [oeyoews@d2123fe](https://github.com/oeyoews/nextjs-notion-starter-kit/commit/d2123fe5ae8ddc9cc7f066f110535b63075481fe) I tested it, it doesn't seem to work

I tested it too, I think a new function should be rewritten to make that work

maybe in future when I learned Javascript and NodeJs

thank you so far

**Comment by oeyoews** [comment]


**Reply by alerezaaa** [reply]
> https://github.com/oeyoews/neoblog/blob/3187ecd7b69103393bbdd0761c611ad6ec4d0713/pages/feed.tsx#L66

I think your repository is private
![image](https://user-images.githubusercontent.com/86428901/208171973-bd63a75b-2d64-426a-a172-fe98346697d7.png)

**Comment by JIeJaitt** [comment]
> Hi there I use "relative date" instead on "month and time" in my Notion database but it is shown as full-date in this theme How can I change the theme codes to show relative date for posts date? I think its related to this line of code: [link](https://github.com/alerezaaa/nextjs-notion-starter-kit/blob/afbab1917fd793eb08729da56700e541ce3b4ee1/components/NotionPage.tsx#L108) But I dont know how to solve it Thanks in advance

Hi, Excuse me, do you know how to achieve this function? No matter what I try I can't get this to work！
<img width="874" alt="image" src="https://user-images.githubusercontent.com/77219045/208232078-047b507b-0b30-463d-95d5-876f0316dfdd.png">

**Reply by alerezaaa** [reply]
> > Hi there I use "relative date" instead on "month and time" in my Notion database but it is shown as full-date in this theme How can I change the theme codes to show relative date for posts date? I think its related to this line of code: [link](https://github.com/alerezaaa/nextjs-notion-starter-kit/blob/afbab1917fd793eb08729da56700e541ce3b4ee1/components/NotionPage.tsx#L108) But I dont know how to solve it Thanks in advance
> 
> Hi, Excuse me, do you know how to achieve this function? No matter what I try I can't get this to work！ <img alt="image" width="874" src="https://user-images.githubusercontent.com/77219045/208232078-047b507b-0b30-463d-95d5-876f0316dfdd.png">

Hi,

Not yet, discussion is going on
